add derivative info (#133)
DCAN-Labs/abcd-hcp-pipeline: docs/outputs.md

# Pipeline Outputs

## Raw Outputs

The raw outputs are organized in the following structure (not all folders are listed here): 

    output_dir/
    └── sub-<SUBID>/
        └── ses-<SESID>/
            ├── files/
            │   ├── DCANBOLDProc_<ver>/
            │   ├── summary_DCANBOLDProc_<ver>/
            │   │   └── executivesummary/
            │   |        └── executive_summary_sub-<SUBID>_ses-<SESID>.html
            │   ├── MNINonLinear/
            │   │   ├── Results/

>             │   │   ├── fsaverage_LR32K/

Reviewer: ```suggestion
            │   │   ├── fsaverage_LR32k/
```

* `T2w`: If a T2w for the subject was provided, this folder will store the native space data.
* `ses-<SESID>_task-<taskname>_run-<num>`: These folders contain intermediate functional preprocessing files from the fMRIVolume stage.

### logs

`logs` contains folders for each stage that contain the log files for that stage. In the case of an error, consult these files *in addition* to the standard error and standard output of the app itself (by default this is printed to the command line).

`status.json` codes:

* unchecked: 999
* succeeded: 1
* incomplete: 2
* failed: 3
* not_started: 4

## BIDS Derivatives

Pipeline outputs are mapped into BIDS-compliant derivatives via [File-mapper](https://github.com/DCAN-Labs/file-mapper) with the following directory structure. A detailed explanation of each file type is provided in the sections below - see anat/ and func/ sections. The `<SUBID>`, `<SESID>`, and `<TASK>` placeholders are replaced with the subject ID, session ID, and task name respectively (e.g. task-rest). The `<T1w|T2w>` placeholder is replaced with the type of anatomical image (T1-weighted or T2-weighted). The `<label>` placeholder is replaced with the run number.

> The `<ATLAS>` placeholder is replaced with the name of the atlas used for functional data for extraction of parcellated timeseries, including Evan Gordon’s 333 ROI AT (Gordon et al., 2014), HCP’s 360 ROI AT (Glasser et al., 2016), Jonathan Power’s 264 ROI AT (Power et al., 2011), and Thomas Yeo’s 118 ROI AT (Yeo et al., 2011). All parcellations additionally include 19 individualized subcortical parcellations.

Reviewer: ```suggestion
The `<ATLAS>` placeholder is replaced with the name of the atlas used for functional data for extraction of parcellated timeseries, including Evan Gordon’s 333 ROI AT (Gordon et al., 2014), HCP’s 360 ROI AT (Glasser et al., 2016), Jonathan Power’s 264 ROI AT (Power et al., 2011), and Thomas Yeo’s 118 ROI AT (Yeo et al., 2011). All parcellations additionally include a subcortical segmentation with ROIs taken from the standard HCP 91k-grayordinate CIFTI space. 
```
```suggestion
The `<ATLAS>` placeholder is replaced with the name of the atlas used for functional data for extraction of parcellated timeseries, including Evan Gordon’s 333 ROI AT (Gordon et al., 2014), HCP’s 360 ROI AT (Glasser et al., 2016), Jonathan Power’s 264 ROI AT (Power et al., 2011), and Thomas Yeo’s 118 ROI AT (Yeo et al., 2011). All parcellations additionally include 19 individualized subcortical parcellations.
```

| `*_<T1w/T2w>_space-MNI_<brain/head>.nii.gz`                                            | T1w & T2w brain & head images in MNI space                                                    |
| `*_T1w_space-MNI_desc-wmparc_dseg.nii.gz`                                              | White matter segmentation in MNI space                                                      |
| `*_hemi-<L/R>_space-<MNI/T1w>_mesh-<fsLR32k/fsLR164k/native>_midthickness.surf.gii`    | L/R midthickness in MNI & native space each with 32k, 146k, * native mesh                                   |
| `*_atlas-MNI_space-fsLR32k(_desc-smoothed)_myelinmap.dscalar.nii`                      | Smoothed & unsmoothed myelin map (only if T2w present)                                          |
| `*_space-ACPC_dseg.nii.gz`                                                             | Discrete segmentation (native volume space)                                                              |
| `*_space-fsLR32k_<curv/sulc/thickness>.dscalar.nii`                                    | Dense curvature, sulcal depth, & cortical thickness                                                     |

> | `*_space-fsLR32k_sulc.pscalar.nii`                                                     | Parcellated dense subject sulcal depth                                                                           |

Reviewer: ```suggestion
| `*_space-fsLR32k_sulc.pscalar.nii`                                                     | Parcellated dense subject sulcal convexity                                                                           |
```

> | `*_space-fsLR32k_sulc.pscalar.nii`                                                     | Parcellated dense subject sulcal depth                                                                           |

Reviewer: ```suggestion
| `*_space-fsLR32k_sulc.pscalar.nii`                                                     | Parcellated dense subject sulcal convexity                                                                           |
```

                                                              

### func/

The derivative files provided under `func/` including the following (`sub-<label>_ses-<label>_` is replaced with an asterisk `*_` for readability):

|            **Filename**                                                                     |         **Description**                                                                                                                |
| ----------------------------------------------------------------------- | ----------------------------------------------- |                                         

> | `*_task-<MID/nback/rest/SST>_bold_desc-filtered_timeseries.dtseries.nii`               | Concatenated functional task dense time series post-DANBOLDProc (regression and filtering) in Atlas space              |

Reviewer: ```suggestion
| `*_task-<MID/nback/rest/SST>_bold_desc-filtered_timeseries.dtseries.nii`               | Concatenated functional task dense time series post-DCANBOLDProc (regression and filtering) in HCP standard 91k CIFTI space           |
```
